src/Service/Azure.DataApiBuilder.Service.csproj

```diff
@@ -10,6 +10,10 @@
         <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
```

Comment:
Do you know what TF_BUILD stands for?

Reply:
`TF_BUILD` is a system variable that gets set to `true` for CI runs ( [Docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml) ). Deterministic builds require that the property `ContinousIntegrationBuild` to be set to `true` during CI runs. Since, `TF_BUILD` turns to be `true` only during pipeline runs, this serves as a way to distinguish between a pipeline run vs local run